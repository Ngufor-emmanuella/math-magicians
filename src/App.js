import React from 'react';
import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
import Calc from './components/calculator';
import NotFoundPage from './components/ErrorPage';
import Home from './components/Home';
import Quotes from './components/Quotes';
import Navigation from './components/Navigation'

function App() {
  return (
    <Router>
      <Navigation />
      <Routes>
      <Route path="/Home" element={<Home />} />
        <Route path="/calculator" element={<Calc />} />
        <Route path="/Quotes" element={<Quotes />} />
        <Route path="*" element={<NotFoundPage />} />
      </Routes>
    </Router>
  );
};

export default App;
