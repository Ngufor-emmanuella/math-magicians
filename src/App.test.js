import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('Jest snapshot to test components', () => {
  const component = renderer.create(
    <App />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
