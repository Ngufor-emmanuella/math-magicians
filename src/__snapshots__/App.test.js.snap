// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest snapshot to test components 1`] = `
Array [
  <div
    className="nav-bar"
  >
    <div
      className="logo"
    >
      <h1
        className="magicians"
      >
        Math Magicians
      </h1>
    </div>
    <nav
      className="nav"
    >
      <a
        className="links"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      |
      <a
        className="links"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      |
      <a
        className="links"
        href="/Quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </nav>
  </div>,
  <main>
    <h2
      className="welcome"
    >
      Welcome to our page
    </h2>
    <div
      className="main-box"
    >
      <p
        className="sub-box"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur. Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.
      </p>
      <p
        className="lores"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.
      </p>
    </div>
  </main>,
]
`;
